modules/core/src/main/java/org/apache/ignite/internal/processors/platform/client/cache/ClientCachePartitionsRequest.java

```diff
@@ -77,14 +75,13 @@ public ClientCachePartitionsRequest(BinaryRawReader reader) {
             if (cacheDesc == null)
                 continue;
 
-            ClientCachePartitionAwarenessGroup grp = processCache(ctx, groups, cacheGroupIds, affinityVer, cacheDesc);
+            ClientCachePartitionAwarenessGroup grp = processCache(ctx, affinityVer, cacheDesc);
```

Comment:
One cache group can contains a lot of caches, but the same mapping. Comparing mapping for each cache is too resource consuming, we should group these comparing by cache groups.

Reply:
Fixed.